### PR TITLE
Task 151437 init deploy

### PR DESCRIPTION
**Pull request by vlegakis-trasys**
Init deployment only changing the FE version number

### CODE REVIEW

File: frontend-service/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eea-reportnet-frontend",
-  "version": "3.0.0",
```
Comment (edual16) [inline frontend-service/package.json:3]: We don't change this version number, this is not the microservice version.

The version from all microservices are in parent-poms/parent/pom.xml.

**Review by edual16 — CHANGES_REQUESTED** [review]

File: frontend-service/package.json (hunk 1 above)
Comment (vlegakis-trasys) [reply frontend-service/package.json:3]: Thanks @edual16 good to know! So the FE doesn't follow any versioning in the package.json? 
Either way this is intended just to get used to the process. I will do some other dummy change and come back to you.

Comment (alfaredu) [reply frontend-service/package.json:3]: One example: the bulkDeleteCheckTooltipShift message could be changed from
Hold Shift+Click to select more than one field
to 
Hold shift to select more than one field
Or something similar

File: frontend-service/src/_styles/_cssVariables.scss [hunk 2]
```diff
@@ -15,7 +15,7 @@
   --c-yellow-50: #ffe879;
   --c-yellow-25: #fff4bb;
   --c-black-color: #000000;
-  --c-gray-110: #404040;
+  --c-gray-110: #404041;
```
Comment (vlegakis-trasys) [inline frontend-service/src/_styles/_cssVariables.scss:18]: @alfaredu I have made this slight change in order to write a test which will catch that

**Review by vlegakis-trasys — COMMENTED** [review]
@alfaredu I have reverted the package.json version, while bumping the version in pom.xml 
Also, I made a small text correction + a slight color change which we'd like to catch in a test. Please take a look and let me know if we could deploy that or what else may be taken into account. Many thanks!

File: frontend-service/package.json (hunk 1 above)
Comment (vlegakis-trasys) [reply frontend-service/package.json]: Done @alfaredu please take a look at my comments on what I've done. I'd like to have a short call if you want at some point about the application overview. Will ping you in Teams. Many thanks!

**Comment by alfaredu** [comment]
We are working on a hotfix so we are not able to merge this PR until we finish it. In the moment you start with changes in this repo we should stop in our repo to avoid conflicts and issues.

**Pull request closed without merging**